src/main/java/step3/study/domain/Drivers.java

```diff
@@ -0,0 +1,54 @@
+package step3.study.domain;
+
+import step3.study.dto.RequestRacingDTO;
+import step3.study.dto.ResponseRacingDTO;
+import step3.study.util.RandomGenerator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Drivers {
+    private final List<Driver> drivers;
+
+    public Drivers(List<Driver> drivers) {
+        this.drivers = drivers;
+    }
+
+    public static List<Driver> of(RequestRacingDTO requestRacingDTO) {
+        String[] driverNames = requestRacingDTO.getDriverNames();

```

Comment:
```suggestion
    public static List<Driver> of(RequestRacingDTO requestRacingDTO) {
        List<String> driverNames = requestRacingDTO.getDriverNames();
```

취향차이라고 생각할 수도 있지만, Collections 를 더 적극적으로 활용하기 위해서라도 List 인터페이스를 사용해주시기 바랍니다.